server/index.js

```diff
@@ -1,14 +1,23 @@
 const express = require('express');
 const app = express();
-const PORT = 3000;
+const bodyParser = require('body-parser');
```

Comment:
Just a style suggestion. Group all your requires together and order them by:
Third-party, app-specific

The 'const app = express()' should be after all the requires. 

Doing all this makes it easy to see what is used and where it's coming from.